async_upnp_client/profiles/profile.py

```diff
@@ -431,8 +431,25 @@ async def _async_poll_state_variables(
         changed_state_variables: List[UpnpStateVariable] = []
 
         for action_name in action_names:
-            action = service.action(action_name)
-            result = await action.async_call(**in_args)
+            try:
+                action = service.action(action_name)
+            except KeyError:
+                _LOGGER.debug(
+                    "Can't poll missing action %s:%s for state variables",
+                    service_name,
+                    action_name,
+                )
+                continue
+            try:
+                result = await action.async_call(**in_args)
+            except UpnpResponseError as err:
+                _LOGGER.debug(
+                    "Failed to call action %s:%s for state variables: %r",
+                    service_name,
+                    action_name,
+                    err,
+                )
+                continue
 
             for arg in action.arguments:
```

Comment:
Won't this fail here, when no `action` can be found?

Reply:
I think in that case the `continue` at line 442 should move onto the next `action`, never reaching here.

Reply:
You are right.